kettle/update.py

```diff
@@ -74,7 +74,7 @@ def main():
     else:
         call(f'{mj_cmd} | pv | gzip > build_staging.json.gz')
         call(f'{bq_cmd} k8s-gubernator:build.staging build_staging.json.gz schema.json')
-        call('python3 stream.py --poll kubernetes-jenkins/gcs-changes/kettle ' \
+        call('python3 stream.py --poll kubernetes-jenkins/gcs-changes/kettle-staging ' \
```

Comment:
won't this change also affect prod?

The subscription name should ideally be a parameter

Reply:
There are two subs now, so it will not. They should not cross contaminate.

Reply:
But i can make it a param but there should really be one per instance.

Reply:
I mean if we make the hardcoded code change, whenever we deploy to prod next it will also pick up the staging subscription instead right ?

Reply:
Prod has it's own call to stream on line 72, which uses the unchanged subscription. So should be 100% fine. Only staging will use the experimental subscription.

Reply:
Agreed, will update as params